finalPrep/modZeroHero.js

```diff
@@ -56,29 +104,95 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
 
+var scaryMonster = {
+  name : "Evilboy",
+  smell : "Evil",
+  weightInLbs : 700,
+  citiesDestroyed : ["Denver", "Dallas", "Houston"],
+  luckyNumbers : [17, 27, 7],
+  address : [17, "Wallaby Way", "Texas", 75002]
+  }
+
+  console.log(scaryMonster);
+
+
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+
+class SuperHero {
+  constructor(name, superpower, age, archNemesis, powerLevel, energyLevel){
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this.archNemesis = "The Syntax Error";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
+
+  sayName(){
+    console.log(this.name);
+  }
+
+  maximizeEnergy(){
+    console.log(this.energyLevel = 1000);
```

Comment:
Since the specification didn't call for you to print anything, you'll want to omit to `console.log` and just have `this.energyLevel = 1000;` inside the method.